### PR TITLE
Add encoding example

### PR DESCRIPTION
**Pull request by latentgenius**

### CODE REVIEW

**Comment by emersion** [comment]
Hi, thanks for contributing! 

Examples in Go are supposed to be self-contained and minimal. In particular, custom data structures (like your `Contact`) won't be shown in GoDoc. Can you change that please?

Also, you can use `card.Set(key, field)` if you set one field instead of `card[key] = []*vcard.Field{field}`. There's also `card.SetValue(key, value)` if you don't want to set params.

**Comment by latentgenius** [comment]
Thanks for the feedback. I've made some changes to the file.

**Comment by latentgenius** [comment]
I noticed there aren't comments in the example_test.go file, except the ones I added. Should I remove the comments?

File: example_test.go [hunk 1]
```diff
@@ -27,3 +29,63 @@ func ExampleNewDecoder() {
...
+		{"Eric", "E.", "Peter", "(+37) 221 9903"},
+		{"Nelson", "D.", "Patrick", "(+1) 122 8810"},
+	}
+
+	var (
+		// card is a map of strings to []*vcard.Field objects
+		card vcard.Card
+
+		// buffer the writes to save disk I/O time
+		writeBuffer = bufio.NewWriter(destFile)
```
Comment (emersion) [inline example_test.go]: Since this is an example, you can elude this. Just write directly to the file, that reduces the noise.

**Comment by emersion** [comment]
Apart from the little comment above, LGTM :)

**Comment by latentgenius** [comment]
How is it now? I removed those lines.

**Comment by emersion** [comment]
Thanks!

**Pull request merged**